lib/sqlalchemy/dialects/mysql/base.py

```diff
@@ -2049,6 +2049,19 @@ def visit_set_column_comment(self, create, **kw):
         )
 
 
+class MariaDBDDLCompiler(MySQLDDLCompiler):
+    def get_column_specification(self, column, **kw):
+        """Builds column DDL."""
+        if column.computed is not None:
+            # NOTE: Generated Column Does not support nullable to be False
+            #       so mark it as True
+            # TODO: Should raise a warning if user explicitly sets value
+            #       as False?
+            column.nullable = True
```

Comment:
I think we should change the value only if the user did not specify one itself. 

there is `_user_defined_nullable` on the column that we can look.

Reply:
Updated as per comments, realised `is_mariadb` flag was available so using instead of creating seperate mariadbDDLcompiler class.
.
updated issue number in commit message